src/index.css

```diff
@@ -0,0 +1,227 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: white;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.logo img {
+  height: 40px;
+}
```

Comment:
The class name `.logo img` does not fully adhere to the BEM naming convention. Consider renaming it to something like `.logo__image` to maintain consistency with BEM principles.